acq4/util/imaging/record_thread.py

```diff
@@ -75,11 +73,18 @@ def recording(self):
         """
         return self._recording
 
-    def saveFrame(self):
-        """Ask the recording thread to save the most recently acquired frame.
-        """
+    def saveFrame(self, bgCtrl, contrastCtrl):
```

Comment:
Get metadata from bg/contrast controls at this point; don't let background threads access widgets